.flake8

```
@@ -6,5 +6,7 @@ ignore=
     W503,
     # line break after binary operator
     W504,
+    # whitespace before ':' https://github.com/psf/black/issues/1437
+    E203
```

let's keep the trailing comma :)

```suggestion
    E203,
```

---

Fixed